### ISSUE TITLE
Support of Multi-model Endpoint using HuggingFace inference toolkit

### ISSUE THREAD

**Issue by Tarun02** [aws/sagemaker-huggingface-inference-toolkit#59]
Hi,

Does SageMaker huggingface inference toolkit supports multi-model endpoint ?

**Comment by philschmid** [comment]
Yes, the HuggingFace Inference Toolkit uses similar mechanism than the other ones. You can find documentation here: 
* https://docs.aws.amazon.com/sagemaker/latest/dg/create-multi-model-endpoint.html
* https://sagemaker.readthedocs.io/en/stable/api/inference/multi_data_model.html

**Comment by DevBey** [comment]
Hi @philschmi,

thanks for the resources !!
in this https://www.philschmid.de/sagemaker-huggingface-multi-container-endpoint you mention that multi-container endpoints are not possible on GPU ??

is this true for multi-modal endpoint also ??

**Comment by philschmid** [comment]
Yes, currently Multi-Model Endpoint also only works with CPU instances

**Comment by DevBey** [comment]
thanks @philschmid  you saved me sometime

**Comment by ibicdev** [comment]
Second this question.  It has been 1.5 years since the original question, yet is GPU still not supported?
